.storybook/webpack.config.js

```diff
@@ -11,5 +11,18 @@ module.exports = ({ config }) => {
   config.resolve.alias["core-js"] = path.dirname(require.resolve("core-js"));
   config.module.rules[0].use[0].options.sourceType = "unambiguous";
 
+  // Taken from stackoverflow
+  // https://stackoverflow.com/a/59639736/3058839
```

Comment:
Thanks for adding links for more info here and in the Jest config. That's really helpful!